yt/yql/providers/yt/gateway/native/yql_yt_native.cpp

```diff
@@ -4875,6 +4879,23 @@ class TYtNativeGateway : public IYtGateway {
         });
     }
 
+    TFuture<void> DoCreateView(TYtCreateView create, const TExecContext<TRunOptions>::TPtr& execCtx) {
```

Comment:
We should support view creation in file provider also. Please see tests in
yt/yql/tests/sql/suites/view. File provider already can read both embedded and standalone views